codalab/model/worker_model.py

```diff
@@ -451,3 +456,32 @@ def has_reply_permission(self, user_id, worker_id, socket_id):
             if row:
                 return True
             return False
+
+    def send_json(
```

Comment:
Maybe rename to `send_json_message` for consistency with `send_json_message_with_sock`

Reply:
Done.